demoapp/src/main/java/com/cube/fusion/android/demoapp/activity/ContentActivityImpl.kt

```diff
@@ -57,15 +58,17 @@ class ContentActivityImpl : FusionContentActivity() {
 		val localSource = AssetsPageSource(this, { it }, resolvers.values)
 		return AndroidFusionConfig(
 			populator = RetrofitDisplayPopulator(this::lifecycleScope, baseUrl, resolvers.values, localSource),
-			actionHandler = DefaultActivityActionHandlers { view, action ->
-				getIntent(view.context, baseUrl, action.extractClick())
-			}.apply {
-				registerNativeActionForLink(NATIVE_ACTION_KEY) { view, _ ->
-					view.context.startActivity(Intent(Settings.ACTION_WIFI_SETTINGS))
-				}
-			},
-			imageLoader = PicassoImageLoader,
-			resolvers = resolvers
+			resolvers = resolvers,
+			viewConfig = AndroidFusionViewConfig(
+				actionHandler = DefaultActivityActionHandlers { view, action ->
+					getIntent(view.context, baseUrl, action.extractClick())
+				}.apply {
+					registerNativeActionForLink(NATIVE_ACTION_KEY) { view, _ ->
+						view.context.startActivity(Intent(Settings.ACTION_WIFI_SETTINGS))
+					}
+				},
+				imageLoader = PicassoImageLoader,
```

Comment:
Should this be Coil now?

Reply:
I can update the demo app to use Coil, sure :)